lib/vagrant-managed-servers/action/sync_folders.rb

```diff
@@ -62,6 +62,10 @@ def call(env)
                                 :hostpath => hostpath,
                                 :guestpath => guestpath))
 
+            # Escape backlashes in case the user is trying to authenticate using
+            # Active Directory domain credentials (e.g. "domain\username")
+            ssh_info.gsub!("\\", "\\\\\\\\")
```

Comment:
You are absolutely right. I did it once in a branch and then remembered that I needed to update my fork and opted to just redo the changes. Update coming.